source/_integrations/airzone_cloud.markdown

```diff
@@ -0,0 +1,39 @@
+---
+title: Airzone Cloud
+description: Instructions on how to integrate Airzone Cloud within Home Assistant.
+ha_release: 2023.6
+ha_category:
+  - Sensor
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_domain: airzone_cloud
+ha_platforms:
+  - sensor
+ha_codeowners:
+  - '@Noltari'
+ha_integration_type: integration
+---
+
+This integration interacts with the Cloud API of [Airzone HVAC zoning systems](https://www.airzone.es/en/).
+
+A typical Airzone device has a *parent zone* (Master Thermostat) per HVAC system, which is the only zone where the HVAC mode can be changed. The rest are *child zones* which can only enable or disable the HVAC and adjust the desired temperature on that specific zone.
+
+Note that multiple HVAC systems can be connected to the same Airzone WebServer. In this case, there will be a *parent zone* per HVAC system and there may also be *child zones* for each HVAC system.
```

Comment:
```suggestion
Note that multiple HVAC systems can be connected to the same Airzone web server. In this case, there will be one *parent zone* per HVAC system and there may also be *child zones* for each HVAC system.
```
Unless web server is part of a product name, there is no need to capitalize it. We try to use [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/capitalization) for common nouns.